package.json

```diff
@@ -248,16 +280,30 @@
           "name": "",
           "icon": "./media/images/dbt_icon.svg"
         }
+      ],
+      "insights_view": [
+        {
+          "id": "dbtPowerUser.Insights",
+          "name": "",
+          "type": "webview"
+        }
       ]
     },
     "commands": [
       {
         "command": "dbtPowerUser.puQuickPick",
+        "category": "dbt Power User",
```

Comment:
added `category` for all the commands to show under `dbt Power User` label in `cmd+shift+P` menu